README.md

```diff
@@ -106,16 +106,16 @@ See the [general partition](/docs/models/shared/partitionparameters.md) page for
 
 #### Splitting PDF by pages
 
-In order to speed up processing of long PDF files, set `split_pdf_page=True`. It will cause the PDF
-to be split page-by-page at client side, before sending to API, and combining individual responses
+In order to speed up processing of long PDF files, `split_pdf_page=True` by default. It will cause the PDF
+to be split at client side, before sending to API, and combining individual responses
 as single result. This will work only for PDF files, so don't set it for other filetypes.
```

Comment:
Hmm the first sentence is a bit ungrammatical. Also, the last warning "don't set it for other filetypes" is a bit scary, because if by default we set it to true, then we shouldn't demand that user disables this flag. Maybe change of tone to "this will affect only PDF files, no need to disable it for other filetypes."

Reply:
description updated